test/generate_sentence_permutations.js

```diff
@@ -18,4 +18,10 @@ describe('Generating all permutation of sentences', () => {
     const sentences = generator.generateSentences(wordList);
     expect(sentences).to.equal(null);
   });
+
+  it('should return empty list', async () => {
```

Comment:
I think we should change it to `should return empty list for null input`
As above test name is also name, it will be easy to differentiate

Reply:
I hope PR #30 addresses this!

Reply:
Why have you created a new PR.
You can make the fix on this branch `add-new-test` it will update this PR itself.
If you will keep creating PR for every review, it will be difficult to track the review and it will just be generating junk of PR's

Reply:
Can you update this PR itself. and close the new one which you have created.